draft-ietf-taps-interface.md

```diff
@@ -2540,6 +2540,41 @@ This document describes a generic API for interacting with a transport services
 Part of this API includes configuration details for transport security protocols, as discussed
 in {{security-parameters}}. It does not recommend use (or disuse) of specific
 algorithms or protocols. Any API-compatible transport security protocol should work in a TAPS system.
+Security consideration for these protocols should be discussed in the respective specifications.
+
+The desribed API is used to exchange information between an application and the transport system. While
+it is not necessarily expected that both systems are implemented by the same authority, it is expected
+that the transport system implementation is either provided as library that is selected by the application
+from a trusted party, or that it is part of the operating system that the application also relies on for
+other tasks.
+
+In any case the TAPS API is an internal interface that is used to change information locally between two systems.
+However, as the transport system is responsible for network communication, it is in the position to
+potentially share any information provided by the application with the network or another communication peer. 
+Most of the information provided over the TAPS API are useful to configure and select protocols and paths
+and are not necessarily privacy sensitive. Still, there is some information that could be privacy sensitve as
+it might reveal usage characteristics and habits of the user of an application. 
+
+It should first be noted that of course any communication over a network reveals usage characteristics, as all
+packets as well as their timing and size are part of the network-visible wire image {{?RFC8546}}. However, of
+course the selection of a protocol and its configuration impacts which information is visible, potentially in
+clear text, to which other enties. In most cases information that is provided for protocol and path selection
```

Comment:
/to which other entities/
- seem to read wrong.. is this: and the other entities that can access this, or something else?